.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: node_js
 node_js:
-    - "6"
-    - "node"
+- 7 # Current stable
```

Comment:
right now `node` = 7, when 8 comes out, 7 is deprecated`.
as `node` matches the latest version and needs less maintenance